ethcore/src/client/client.rs

```diff
@@ -648,7 +648,7 @@ impl BlockChainClient for Client {
 			timestamp: view.timestamp(),
 			difficulty: view.difficulty(),
 			last_hashes: last_hashes,
-			gas_used: U256::zero(),
+			gas_used: Default::default(),
```

Comment:
yes i think U256::zero() is much more preferred even
we deal with numbers here, not with some objects/classes